src/manifest.json

```
@@ -1,12 +1,12 @@
 {
 	"name": "Web Maker",
-	"version": "4.0.3",
+	"version": "4.1.0",
```

```suggestion
	"version": "4.0.3",
```

version would be automatically bumped